static/workspace-view.less

```diff
@@ -22,7 +22,9 @@ atom-workspace {
   position: relative;
   color: @text-color;
   background-color: @app-background-color;
-  font-family: @font-family;
+  font: menu;
```

Comment:
I think we can remove all the font properties here, since they are already set on `html`. I guess it got initially added because Bootstrap added a font to `body`, but that is now overridden https://github.com/atom/atom/blob/master/static/bootstrap-overrides.less#L25-L28, so it's not needed here anymore.